Whiskey/en-US/about_Whiskey_PublishPowerShellModule_Task.help.txt

```diff
@@ -9,20 +9,28 @@ SUMMARY
 DESCRIPTION
     The `PublishPowerShellModule` task publishes a PowerShell module to a NuGet-based PowerShell repository (e.g. the PowerShell Gallery, ProGet, etc.). It uses the `Publish-Module` function from the `PowerShellGet` module.
     
-    You must supply the the path to the module's directory with the `Path` property, the URI of the repository with the `RepositoryUri` property, the name of the repository with the `RepositoryName` property, and the ID of the API key to use with the `ApiKeyID`.
+    You must supply the the path to the module's directory with the `Path` property, the name of the repository with the `RepositoryName` property, and the ID of the API key to use with the `ApiKeyID`.
 
-    If a repository named `RepositoryName`, doesn't exist, one is registered and its source and publish locations are both set to `RepositoryUri` (using the `PowerShellGet` modules's `Register-PSRepository` function.
+    If a repository named `RepositoryName` doesn't exist, you must supply the `RepositoryUri` parameter so that the task can register the repository. The `RepositoryUri` parameter is passed to the `Register-PSRepository` function's `PublishLocation` and `SourceLocation` parameters.
 
-    Before publishing, the `ModuleVersion` property in the module manifest is updated to be the version of the current build, minus any prerelease or build metadata.
+    Before publishing, the `ModuleVersion` property in the module manifest is updated to be the version of the current build, minus any prerelease or build metadata and the `Prerelease` property in the `PrivateData.PSData` section is updated to be the current prerelease metadata. Make sure you use prerelease metadata in the correct semver format for the repository to which you're publishing. The task uses regular expressions to find and replace the module version and prerelease. It looks for (excluding the comments):

```

Comment:
Should we offer a way to disable this functionality? Sometimes people write their own version automation or use tools that do automated version management during builds, which this functionality would interfere with.

Reply:
Good idea. I entered issue https://github.com/webmd-health-services/Whiskey/issues/239 to get this functionality added.